_viash.yaml

```diff
@@ -0,0 +1,15 @@
+viash_version: 0.9.0
+
+source: src
+target: target
+
+config_mods: |
+  .functionality.version := 'dev'
+  .functionality.arguments[.multiple == true].multiple_sep := ';'
+  .platforms[.type == 'docker'].target_registry := 'ghcr.io'
+  .platforms[.type == 'docker'].target_organization := 'openproblems-bio/task-template'
+  .platforms[.type == 'docker'].target_image_source := 'https://github.com/openproblems-bio/task-template'
+  .platforms[.type == "nextflow"].directives.tag := "$id"
+  .platforms[.type == "nextflow"].auto.simplifyOutput := false
+  .platforms[.type == "nextflow"].config.labels := { lowmem : "memory = 20.Gb", midmem : "memory = 50.Gb", highmem : "memory = 100.Gb", lowcpu : "cpus = 5", midcpu : "cpus = 15", highcpu : "cpus = 30", lowtime : "time = 1.h", midtime : "time = 4.h", hightime : "time = 8.h", veryhightime : "time = 24.h" }
+  .platforms[.type == "nextflow"].config.script := "process.errorStrategy = 'ignore'"
```

Comment:
Make use of the new fields added in Viash 0.9. Example: https://github.com/openproblems-bio/task_cell_cell_communication/blob/main/_viash.yaml